DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/NetworkProtectionTunnelController.swift

```diff
@@ -732,54 +730,25 @@ final class NetworkProtectionTunnelController: TunnelController, TunnelSessionPr
         try await tunnelManager.saveToPreferences()
     }
 
-    /* Temporarily disabled until we fix this menu: https://app.asana.com/0/0/1205766100762904/f
-    @MainActor
-    private func excludedRoutes() -> [NetworkProtection.IPAddressRange] {
-        settings.excludedRoutes.compactMap { [excludedRoutesPreferences] item -> NetworkProtection.IPAddressRange? in
-            guard case .exclusion(range: let range, description: _, default: let defaultValue) = item,
-                  excludedRoutesPreferences[range.stringRepresentation, default: defaultValue] == true
-            else { return nil }
-            // TO BE fixed:
-            // when 10.11.12.1 DNS is used 10.0.0.0/8 should be included (not excluded)
-            // but marking 10.11.12.1 as an Included Route breaks tunnel (probably these routes are conflicting)
-            if settings.enforceRoutes && range == "10.0.0.0/8" {
-                return nil
-            }
+    // MARK: - Routing
 
-            return range
-        }
-    }*/
+    private var enforceRoutes: Bool {
 
-    /// extra Included Routes appended to 0.0.0.0, ::/0 (peers) and interface.addresses
-    @MainActor
-    private func includedRoutes() -> [NetworkProtection.IPAddressRange] {
-        []
-    }
-
-    /* Temporarily disabled - https://app.asana.com/0/0/1205766100762904/f
-    @MainActor
-    func setExcludedRoute(_ route: String, enabled: Bool) {
-        excludedRoutesPreferences[route] = enabled
-    }
-
-    @MainActor
-    func isExcludedRouteEnabled(_ route: String) -> Bool {
-        guard let range = IPAddressRange(from: route),
-              let exclusionListItem = settings.exclusionList.first(where: {
-                  if case .exclusion(range: range, description: _, default: _) = $0 { return true }
-                  return false
-              }),
-              case .exclusion(range: _, description: _, default: let defaultValue) = exclusionListItem else {
-
-            assertionFailure("Invalid route \(route)")
+        guard internalUserDecider.isInternalUser else {
             return false
         }
-        // TO BE fixed: see excludedRoutes()
-        if settings.enforceRoutes && route == "10.0.0.0/8" {
-            return false
+
+        /// Even though there's a remote feature flag, we still want to allow internal users to disable enforceRoutes
+        /// manually in case they have trouble.  To achieve this we force the setting to ON once, but otherwise
+        /// allow the internal user to disable it again.
+        if !settings.enforceRoutesForceEnabledOnce {
+            settings.enforceRoutesForceEnabledOnce = true
+            settings.excludeLocalNetworks = true
+            settings.enforceRoutes = true
```

Comment:
This is basically a one-time enabling of `enforceRoutes` for internal users.  This can be reset in Debug Menu > VPN > Reset > Reset Enforce Routes Enabled Once (or something like that heh).